src/app/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,6 @@
+<app-header></app-header>
+<section class="page-not-found-main">
+  <div class="page-not-found-container">
+    <img src="../../assets/images/page_not_found.png" alt="PageNotFound">
```

Comment:
prefer absolute path for assets `src="src/app/assets/images/page_not_found.png"`